inference-engine/thirdparty/clDNN/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -984,7 +984,7 @@ TEST(activation_f32_fw_gpu, relu_basic_acosh_yxfb) {
     topology topology(
             input_layout("input", input->get_layout()),
             reorder("reorder", "input", input->get_layout().with_padding(padding{{0, 0, 2, 1}, 0})),
-            activation("relu", "reorder", activation_func::acosh, {0.5f, 0.f}, padding{{0, 0, 0, 0}, 0}));
+            activation("relu", "reorder", activation_func::acosh, {0.5f, 0.f}, "", padding{{0, 0, 0, 0}, 0}));
```

Comment:
Please use unified code style as in other tests in this file and add spaces inside figure braces.